nav2_mission_executor/include/nav2_mission_executor/mission_executor.hpp

```diff
@@ -15,21 +15,24 @@
 #ifndef NAV2_MISSION_EXECUTOR__MISSION_EXECUTOR_HPP_
 #define NAV2_MISSION_EXECUTOR__MISSION_EXECUTOR_HPP_
 
+#include <memory>
 #include "nav2_tasks/execute_mission_task.hpp"
 #include "geometry_msgs/msg/pose_stamped.hpp"
 
 namespace nav2_mission_executor
 {
 
-class MissionExecutor : public nav2_tasks::ExecuteMissionTaskServer
+class MissionExecutor : public rclcpp::Node
 {
 public:
   MissionExecutor();
 
   nav2_tasks::TaskStatus execute(
```

Comment:
If this is no longer an implementation of a task, I think the `Execute()` method should be renamed to be more descriptive as its no longer an abstract implementation

Reply:
Sure. I'll update these.